src/components/CaptureMatching/CaptureMatchingFilter.js

```diff
@@ -0,0 +1,236 @@
+
+  // const formatDate = (date) => {
+  //   return convertDateToDefaultSqlDate(date);
+  // };
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    // save the filer to context for editing & submit
+  }
+
+  function handleReset() {
+    // reset form values, except 'approved' and 'active' which we'll keep
+    setDateStart(dateStartDefault);
+    setDateEnd(dateEndDefault);
+    setOrganizationId(ALL_ORGANIZATIONS);
+  }
+
+  const defaultOrgList = userHasOrg
+    ? [
+        {
+          id: ALL_ORGANIZATIONS,
+          name: 'All',
+          value: 'All',
+        },
+      ]
+    : [
+        {
+          id: ALL_ORGANIZATIONS,
+          name: 'All',
+          value: 'All',
+        },
+        {
+          id: ORGANIZATION_NOT_SET,
+          name: 'Not set',
+          value: null,
+        },
+      ];
+
+  return (
+    <>
+      {
+        <form onSubmit={handleSubmit} className={classes.root}>
+          <Grid
+            container
+            wrap="nowrap"
+            direction="column"
+            className={classes.gridContainer}
+          >
+            <Grid item className={classes.inputContainer} direction="column">
+              <MuiPickersUtilsProvider
+                utils={DateFnsUtils}
+                locale={getDatePickerLocale()}
+              >
+                <KeyboardDatePicker
+                  margin="normal"
+                  id="start-date-picker"
+                  htmlFor="start-date-picker"
+                  label="Start Date"
+                  format={getDateFormatLocale(true)}
+                  value={dateStart}
+                  onChange={handleDateStartChange}
+                  maxDate={dateEnd || Date()} // Don't allow selection after today
+                  KeyboardButtonProps={{
+                    'aria-label': 'change date',
+                  }}
+                />
+                <KeyboardDatePicker
+                  margin="normal"
+                  id="end-date-picker"
+                  htmlFor="end-date-picker"
+                  label="End Date"
+                  format={getDateFormatLocale(true)}
+                  value={dateEnd}
+                  onChange={handleDateEndChange}
+                  minDate={dateStart || datePickerDefaultMinDate}
+                  maxDate={Date()} // Don't allow selection after today
+                  KeyboardButtonProps={{
+                    'aria-label': 'change date',
+                  }}
+                />
+              </MuiPickersUtilsProvider>
+              {
+                /* {!userHasOrg && ( }*/
+                <TextField
+                  data-testid="org-dropdown"
+                  select
+                  label="Organization"
+                  htmlFor="organization"
+                  id="organization"
+                  value={organizationId}
+                  onChange={(e) => setOrganizationId(e.target.value)}
+                >
+                  {[...defaultOrgList, ...orgList].map((org) => (
```

Comment:
I'm seeing a warning in the console:
> Material-UI: You have provided an out-of-range value `null` for the select component.
Consider providing a value that matches one of the available options or ''.
The available values are `ALL_ORGANIZATIONS`, `ORGANIZATION_NOT_SET`, `11`, `13`, `14`, `15`, `1`.